judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ItemEssayCard/ItemEssayForm/ItemEssayForm.test.tsx

```diff
@@ -0,0 +1,236 @@
+import { ReactWrapper, mount } from 'enzyme';
+import { ItemEssayFormProps, ItemEssayFormState } from './ItemEssayForm';
+import { ItemType, ItemEssayConfig } from 'modules/api/sandalphon/problemBundle';
+import { AnswerState } from 'components/ProblemWorksheetCard/Bundle/itemStatement';
+import ItemEssayForm from './ItemEssayForm';
+import * as React from 'react';
+
+describe('ItemEssayForm', () => {
+  let wrapper: ReactWrapper<ItemEssayFormProps, ItemEssayFormState>;
+  const itemConfig: ItemEssayConfig = {
+    statement: 'statement',
+    score: 100,
+  };
+
+  beforeEach(() => {
+    window.confirm = jest.fn(() => true);
+  });
+
+  describe('not answered yet', () => {
+    const props: ItemEssayFormProps = {
+      jid: 'jid',
+      type: ItemType.Essay,
+      meta: 'meta',
+      config: itemConfig,
+      disabled: false,
+      onSubmit: jest.fn().mockReturnValue(undefined),
+      answerState: AnswerState.NotAnswered,
+    };
+
+    beforeEach(() => {
+      wrapper = mount(<ItemEssayForm {...props} />);
+    });
+
+    describe('initial condition', () => {
+      it('initial state', () => {
+        const state = wrapper.state();
+        const expectedState = {
+          answerState: props.answerState,
+          answer: '',
+          initialAnswer: '',
+          cancelButtonState: AnswerState.NotAnswered,
+        };
+        expect(state).toEqual(expectedState);
+      });
+
+      it('textarea', () => {
```

Comment:
Please reevaluate test names in this PR. `it()` should be readable as an english sentence. (`It text area` does not make sense)